maint_tools/check_gha_workflow.py

```diff
@@ -77,7 +77,15 @@
 UPDATE_TSV = True
 
 # used by set_python_version to filter jobs by their python version
-EXPECTED_PYTHON_VERSIONS = ["3.8", "3.9", "3.10", "3.11", "3.12", "3.13"]
+EXPECTED_PYTHON_VERSIONS = [
+    "3.8",
```

Comment:
Why do we have P3.8 here ?

Reply:
good question

this maintenance script will be used to check past test runs to see how long they take sorted by OS and python version (see https://nilearn.github.io/dev/ci.html#check-gha-workflow-yml) to plotly graphs like this

![test_runs_timing](https://github.com/user-attachments/assets/4faca755-8f22-4128-8453-217bb836de9f)

the content of really old CI runs (those done with python 3.8) has been purged but their metadata regarding how long they took is still here, so that's why it's still here.